networkx/drawing/layout.py

```diff
@@ -1248,15 +1248,14 @@ def rescale_layout(pos, scale=1):
     --------
     rescale_layout_dict
     """
+    import numpy as np
+
     # Find max length over all dimensions
-    lim = 0  # max coordinate for all axes
-    for i in range(pos.shape[1]):
-        pos[:, i] -= pos[:, i].mean()
-        lim = max(abs(pos[:, i]).max(), lim)
+    pos -= pos.mean(axis=0)
+    lim = max(np.abs(pos).max(), 0)  # max coordinate for all axes
```

Comment:
Given the subsequent `lim > 0`, I don't think the outer `max` is even necessary:

```suggestion
    lim = np.abs(pos).max()  # max coordinate for all axes
```

Reply:
Good point. Will remove 👍